tests/VisualStudio.Roslyn.SDK/Roslyn.SDK.IntegrationTests/InProcess/InProcComponent.cs

```diff
@@ -30,25 +32,30 @@ protected async Task ExecuteCommandAsync(string commandName, string args = "")
         {
             await JoinableTaskFactory.SwitchToMainThreadAsync();
 
-            var dte = await GetGlobalServiceAsync<SDTE, EnvDTE.DTE>();
+            var dte = await GetRequiredGlobalServiceAsync<SDTE, EnvDTE.DTE>();
             dte.ExecuteCommand(commandName, args);
         }
 
-        protected async Task<TInterface> GetGlobalServiceAsync<TService, TInterface>()
+        protected async Task<TInterface> GetRequiredGlobalServiceAsync<TService, TInterface>()
             where TService : class
             where TInterface : class
         {
             await JoinableTaskFactory.SwitchToMainThreadAsync();
 
-            var serviceProvider = (IAsyncServiceProvider2)await AsyncServiceProvider.GlobalProvider.GetServiceAsync(typeof(SAsyncServiceProvider));
-            Assumes.Present(serviceProvider);
-            return (TInterface)await serviceProvider.GetServiceAsync(typeof(TService));
+            var serviceProvider = (IAsyncServiceProvider2?)await AsyncServiceProvider.GlobalProvider.GetServiceAsync(typeof(SAsyncServiceProvider));
+            if (serviceProvider is null)
+            {
+                throw new InvalidOperationException($"Unable to get service {typeof(IAsyncServiceProvider2).FullName}");
+            }
+
+            var @interface = (TInterface?)await serviceProvider.GetServiceAsync(typeof(TService));
+            return @interface ?? throw new InvalidOperationException($"Unable to get service {typeof(TInterface).FullName}");
```

Comment:
```suggestion
            Assumes.Present(@interface);
            return @interface;
```

Reply:
`Assumes.Present` is not null annotated so we'd need to bang it

Reply:
It is in 16.8.

Reply:
ah, I'll upgrade `Microsoft.VisualStudio.Validation`